### ISSUE TITLE
lists not styled in ODF->pdf converter

### ISSUE THREAD

**Issue by GoogleCodeExporter** [dbarra/xdocreport#99]
```
Lists and list items are not styled in ODF->pdf converter.
Style information like font size, indent etc are not processed.

You can see an example in ODTBig.odt. On page 2 there is a list. In resulting 
pdf the list looks different (spans two pages)
```

Original issue reported on code.google.com by `abe...@gmail.com` on 5 Apr 2012 at 8:37

**Comment by GoogleCodeExporter** [comment]
```
Angelo

As you know there is 'fontEncoding' property in PDFViaITextOptions. It's 
default value is platform encoding.
I've changed default font encoding to Element.IDENTITY_H (unicode) and all 
example *.odt files are correcly converted.
So I,m going to leave 'fontEncoding' property but change its default value to 
unicode. What do you think about it?

Regards
Leszek
```

Original comment by `abe...@gmail.com` on 2 May 2012 at 11:57

**Comment by GoogleCodeExporter** [comment]
```
Hi Leszek,

I trust you. I would like test with my computer your update but I cannot do 
that for the momemnt (perhaps at end of this month) 

So modifiy as you wish.

Regards angelo
```

Original comment by `angelo.z...@gmail.com` on 2 May 2012 at 12:05

**Comment by GoogleCodeExporter** [comment]
```
fixed in version 0.9.8
```

Original comment by `abe...@gmail.com` on 18 May 2012 at 11:21
- Changed state: **Fixed**
